.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           command: |
             . venv/bin/activate && mkdir packages
             # build main dash
-            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            git clone -b persistence-hg --depth 1 https://github.com/plotly/dash.git dash-main
```

Comment:
With https://github.com/plotly/dash/pull/1376 now merged, this can be 🔪

Reply:
6df0230